pkg/apis/nmstate/v1/nodenetworkstate_types_test.go

```diff
@@ -88,6 +88,14 @@ status:
 		})
 	})
 
+	Context("when read NodeNetworkState struct from invalid yaml", func() {
+
+		It("should return error", func() {
+			err := yaml.Unmarshal([]byte("plaintext"), &NodeNetworkState{})
```

Comment:
Let's move this to BeforeEach() so the "It" keeps only the Expect, I know it's more verbose.

Reply:
Not sure about that. It() should perform the test, process of the unmashal is part of the test. Let's say it would take 30 seconds to finish unmarshal and either get error or nil. It() should be the one blocking, not beforeEach.

I think we should reorganize these tests a little bit. Keeping test data on one pile outside of tests is IMO not a good practice.

Reply:
Ack, then we reorganize we can put at helpers or something.